nmostesting/NMOSTesting.py

```diff
@@ -151,6 +153,16 @@
 ocsp_app.register_blueprint(OCSP_API)  # OCSP server
 FLASK_APPS.append(ocsp_app)
 
+# Primary Authorization server
+if CONFIG.ENABLE_AUTH and CONFIG.ENABLE_HTTPS:

```

Comment:
In other places, we seem to allow insecure auth for debug purposes? I.e. maybe only check `ENABLE_AUTH` and vary whether HTTP or HTTPS based on `ENABLE_HTTPS`?

Reply:
AUTH should only be used in HTTPS, that is why I was thinking of hardcode all AUTH-related functions using HTTPS only.  I don't mind having HTTP for the debugging purpose, maybe we should add some words in the config saying something like using `ENABLE_AUTH` should have the `ENABLE_HTTPS` set.

Reply:
We should definitely note that in production auth would only be used over secure comms.
Just wondering if one advantage of using a mock Auth server could be that requests/responses can be easily captured and debugged? (Otherwise I think you can do that by e.g. giving WireShark the HTTPS private keys?)

My point is that there seems to be existing code that only checks `ENABLE_AUTH` (or `self.authorization` which is the same) and then uses URLs that depend on `ENABLE_HTTPS`, so that code appears to allow one to try `ENABLE_AUTH = True` with `ENABLE_HTTPS = False`.

Probably either everything should be hardcoded to check both flags, or work in the above mode, consistently?

Reply:
Almost ready to support both HTTP and HTTPS

Reply:
Done